src/layouts/base-new/index.tsx

```diff
@@ -20,7 +22,11 @@ const { ConsentManager, openConsentManager } = createConsentManager({
  * TODO (future enhancement): rename and abstract `SidebarNavDataProvider` for
  * use here.
  */
-const BaseNewLayout = ({ children, showFooter = true }: BaseNewLayoutProps) => {
+const BaseNewLayout = ({
+  children,
+  showFooter = true,
+  showFooterTopBorder = false,
```

Comment:
Could be helpful to write a `TODO` comment re: removing that `showFooter` prop

Reply:
Ah great call - I went to add the `TODO`, and realized it's very safe to flat-out remove `showFooter` since `SidebarSidecarLayout` was the only layout using the prop. So, removed! ✅